types/react-navigation/index.d.ts

```diff
@@ -1254,6 +1254,7 @@ export function createNavigationContainer(
 /**
  * Create an app container to wrap the root navigator
  *
+ * @see https://github.com/react-navigation/react-navigation-native/blob/master/src/createAppContainer.js#L64
```

Comment:
You may want to modify this link to a specific commit id, since line number in the master branch may be changed.

Reply:
Thanks for the input, it's done!